package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshImporter.cs

```diff
@@ -506,13 +512,29 @@ public class GeometrySubsets {
               usdMesh.colors[i] = usdMesh.colors[i];
             }
             unityMesh.colors = usdMesh.colors;
+            forceVertexColors = false;
           } catch (Exception ex) {
             Debug.LogException(
                 new Exception("Error unrolling Face-Varying colors at <" + path + ">", ex));
           }
         } else {
           Debug.LogWarning("Uniform (color per face) display color not supported");
         }
+
+        if (forceVertexColors) {
+            Color [] colors = new Color[usdMesh.points.Length];
+            if (usdMesh.colors.Length != usdMesh.points.Length) {
+              Color color = usdMesh.colors[0].gamma;
```

Comment:
Do you actually want `gamma` here?

I think we only want the color in gamma space when it's used as a material parameter, and now I'm questioning if that's even correct for HDRP/UniversalRP...

Reply:
Good point. I suppose not in this case. Likewise with *RP which are linear, correct?

Reply:
Yeah, I just confirmed the SRP pipelines are linear, so we have a bug in the other code calling .gamma above.